tuned/consts.py

```diff
@@ -122,6 +122,7 @@
 CFG_UNIX_SOCKET_CONNECTIONS_BACKLOG = "connections_backlog"
 CFG_CPU_EPP_FLAG = "hwp_epp"
 CFG_ROLLBACK = "rollback"
+CFG_CUSTOM_PROFILE_DIRS = "custom_profile_dirs"
```

Comment:
As mentioned above, it might be a good idea to change `_dirs` to `_paths` due to consistency with other `tuned-main.conf` options.

Reply:
Here I would disagree. The documentation/manual pages always refer to (custom) profile directories, not paths, so directories seem more clear to me.